app/views/stories/_listdetail.html.erb

```diff
@@ -48,7 +48,7 @@ class="story <%= story.current_upvoted? ? "upvoted" : "" %>
     <% end %>
     <% if story.can_be_seen_by_user?(@user) %>
       <span class="tags">
-        <% story.tags.each do |tag| %>
+        <% (story.previewing ? story.preview_tags : story.tags).each do |tag| %>
```

Comment:
Oh, btw, with this change, starting the server also makes the files `app/views/home/index.html.erb`  and `app/views/search/index.html.erb` change to have this same thing changed. IDK why that happens. Is that OK? Should i commit those changes too?

Reply:
This is `config/initializers/heinous_inline_partial` doing its work. Please do commit those changes so the codebase is in a consistent state with what the server will run with.

Reply:
LOL, that's a poetic name. I've amended those changes onto their appropriate commit!

Reply:
BTW, i also get this change on  `app/views/home/index.html.erb` when rendering `/stories/new`:

```diff
diff --git a/app/views/home/index.html.erb b/app/views/home/index.html.erb
index 4193138f..633531b8 100644
--- a/app/views/home/index.html.erb
+++ b/app/views/home/index.html.erb
@@ -183,7 +183,7 @@ class="story <%= story.current_upvoted? ? "upvoted" : "" %>
           <span> | </span>
           <span class="dropdown_parent">
             <input id="archive_<%= story.short_id %>" class="archive_button" type="checkbox">
-            <label for="archive_<%= story.short_id %>">caches</label>
+            <label for="archive_<%= story.short_id %>">archive</label>
             <div class="archive-dropdown">
               <a href="<%= story.archiveorg_url %>">Archive.org</a>
               <a href="<%= story.archivetoday_url %>">Archive.today</a>
```

But it also happens on `master` so i guess there's some inconsistency on these partial inlines. I'm excluding that change from this PR as it's unrelated.

Reply:
Yes,  e3456c07 missed that update, I'll get it. Living up to its name!